File: 2023 - 3 Semestre/Programação Orientada a Objetos/2023 03 08 - Aula 2/ContaBancaria.cs
public class ContaBancaria {
    public double Saldo { get; set; }
    public int Numero { get; set; }

    public void Depositar(double valor) {
        Saldo += valor;
    }

    public void Sacar(double valor) {
        if (Saldo >= valor) {
            Saldo -= valor;
        }
        else {
            Console.WriteLine("Saldo insuficiente.");
        }
    }
}

File: 2023 - 3 Semestre/Laboratório de Programação/2023 03 31 Aula 5/Exercicio 1/src/ContaBancaria.java
public class ContaBancaria {
    public int numeroDaConta;
    public double saldo;
    public String titular;

    double Depositar(double deposito){
        saldo = saldo + deposito;
        return saldo;
    }
    double Sacar(double saque){
        saldo = saldo - saque;
        return saldo;
    }
    void Transferir(double valor, ContaBancaria conta){
        saldo = saldo - valor;
        conta.saldo += valor;

    }
}
